src/modules/visits/controllers/visits.controller.ts

```diff
@@ -97,11 +93,48 @@ export class VisitsController {
 		@Body() updateVisitDto: UpdateVisitDto,
 		@Session() session: Record<string, any>
 	): Promise<Visit> {
-		const visit = await this.visitsService.update(
-			id,
-			updateVisitDto,
-			SessionHelper.getArchiefUserInfo(session).id
-		);
+		const user = SessionHelper.getArchiefUserInfo(session);
+		const visit = await this.visitsService.update(id, updateVisitDto, user.id);
+
+		if (updateVisitDto.status) {
+			// Status was updated
+
+			// Send notifications
+			const space = await this.spacesService.findById(visit.spaceId);
+			if (updateVisitDto.status === VisitStatus.APPROVED) {
+				await this.notificationsService.create({
+					title: i18n.t('Je aanvraag voor leeszaal {{name}} is goedgekeurd', {
```

Comment:
Is het een idee om de opbouw van de notification in een aparte class te steken of in de notificationsService? dan is het hier wat cleaner. 
Zou hier precies enkel
`await this.notificationsService.visitApproved(visit, space);` schrijven ofzo
Hoeft deze PR niet tegen te houden, maar lijkt me mogelijke improvement voor een volgende

Reply:
ja, klinkt goed